File: hard/18_knapsack_problem/knapsack_problem.test.js
const knapsackProblem = require('./knapsack_problem');

describe('TestProgram', () => {
  it('Test Case 1', () => {
    expect(knapsackProblem([[1, 2], [4, 3], [5, 6], [6, 7]], 10)).toEqual([10, [1, 3]]);
  });
});

File: hard/18_knapsack_problem/knapsack_problem_test.py
import unittest
from knapsack_problem import knapsackProblem

class TestProgram(unittest.TestCase):
    def test_case_1(self):
        self.assertEqual(knapsackProblem([[1, 2], [4, 3], [5, 6], [6, 7]], 10), [10, [1, 3]])
